src/main/resources/l10n/JabRef_en.properties

```diff
@@ -2103,7 +2094,6 @@ List\ must\ not\ be\ empty.=List must not be empty.
 Add\ field\ to\ filter\ list=Add field to filter list
 Add\ formatter\ to\ list=Add formatter to list
 Filter\ List=Filter List
-New\ column=New column
```

Comment:
Oh, this was never used?

Reply:
This was formerly the name for a new column created by an constructor which was never used, as new columns always had by design the name of a field.
This constructor was removed and new columns now always get created by a specified name.